app/code/Magento/Security/Model/AdminSessionsManager.php

```diff
@@ -314,24 +331,27 @@ protected function createAdminSessionInfoCollection()
     }
 
     /**
-     * Calculates diff between now and last session updated_at
-     * and decides whether new prolong must be triggered or not
+     * Calculates diff between now and last session updated_at and decides whether new prolong must be triggered or not
```

Comment:
can you please revert back to the previous implementation here as the line is exceeding 120 characters.